iac-extensions/docker/src/test/resources/checks/PrivilegedUserCheck/Dockerfile_multi_stage_build

```diff
@@ -0,0 +1,5 @@
+FROM gcc as build
+
+# Noncompliant@+1 {{Scratch images run as root by default. Make sure it is safe here.}}
+  FROM scratch
+# ^^^^^^^^^^^^
```

Comment:
Could you add another test file where the first `FROM` is sensitive and the second is not ? I don't think I saw this use case.